src/main/java/gregtech/api/recipes/machines/RecipeMapFormingPress.java

```diff
@@ -23,8 +23,8 @@ public class RecipeMapFormingPress extends RecipeMap<SimpleRecipeBuilder> {
 
     private static ItemStack NAME_MOLD = ItemStack.EMPTY;
 
-    public RecipeMapFormingPress(String unlocalizedName, int minInputs, int maxInputs, int minOutputs, int maxOutputs, int minFluidInputs, int maxFluidInputs, int minFluidOutputs, int maxFluidOutputs, SimpleRecipeBuilder defaultRecipe, boolean isHidden) {
-        super(unlocalizedName, minInputs, maxInputs, minOutputs, maxOutputs, minFluidInputs, maxFluidInputs, minFluidOutputs, maxFluidOutputs, defaultRecipe, isHidden);
+    public RecipeMapFormingPress(String unlocalizedName, int maxInputs, int maxOutputs, int maxFluidInputs, int maxFluidOutputs, SimpleRecipeBuilder defaultRecipe, boolean isHidden) {
```

Comment:
Should these special Recipe Maps get deprecated constructors, like RecipeMap?

Reply:
It is extremely unlikely for an addon mod to be using these recipemaps, so I think it's fine to just remove them.